TicTacToeProject/HumanPlayer.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace TicTacToeApp
+{
+    class HumanPlayer : PlayerInterface
+    {
+        private string _marker;
+
+        public HumanPlayer(string marker)
+        {
+            _marker = marker;
+        }
+
+        public string marker
+        {
+            get
+            {
+                return _marker;
+            }
+            set
+            {
+                _marker = value;
+            }
+        }
+
+        public string getMove(List<int> availableSpaces)
```

Comment:
Unused params are a code smell. Fine to leave it for now to see how this evolves (maybe you end up handing that list of available spaces to the UI) but keep an eye on it. Might need to redesign the shared interface between human and computer players.